library/core/src/alloc/layout.rs

```diff
@@ -503,6 +515,22 @@ impl fmt::Display for LayoutError {
 mod verify {
     use super::*;
 
+    // pub const fn from_size_align(size: usize, align: usize) -> Result<Self, LayoutError>
+    #[kani::proof_for_contract(Layout::from_size_align)]
+    pub fn check_from_size_align() {
+        let s = kani::any::<usize>();
+        let a = kani::any::<usize>();
+
+        if let Ok(layout) = Layout::from_size_align(s, a) {
```

Comment:
Can you implement `kani::Arbitrary` and `kani::Invariant` for Layout instead?

Reply:
See https://github.com/model-checking/verify-rust-std/pull/42#discussion_r1703916363